server/sonar-process-monitor/src/main/java/org/sonar/application/process/SQProcess.java

```diff
@@ -43,7 +43,8 @@
   private final long watcherDelayMs;
 
   private ProcessMonitor process;
-  private StreamGobbler gobbler;
+  private StreamGobbler stdOutGobbler;
+  private StreamGobbler stdErrGobbler;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "stdErrGobbler" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1450)